webpack.mix.js

```diff
@@ -17,6 +17,7 @@ mix
 	.setPublicPath( 'dist' )
 	.scripts( 'node_modules/sharer.js/sharer.js', 'dist/scripts/sharer.js' )
 	.scripts( 'node_modules/lity/dist/lity.js', 'dist/scripts/lity.js' )
+	.scripts( 'assets/src/scripts/jquery.scrollTo.js', 'dist/scripts/jquery.scrollTo.js' )
```

Comment:
See previous comment, then:

```suggestion
.scripts( 'node_modules/jquery.scrollto/jquery.scrollTo.js', 'dist/scripts/jquery.scrollTo.js' )
```

(I think that should be the correct path.)